src/Microsoft.DotNet.XHarness.iOS.Shared/AppBundleInformationParser.cs

```diff
@@ -162,20 +162,35 @@ public async Task<AppBundleInformation> ParseFromAppBundle(string appPackagePath
             bundleExecutable: bundleExecutable);
     }
 
-    private async Task<string> GetPlistProperty(string plistPath, string propertyName, ILog log, CancellationToken cancellationToken = default)
+    private async Task<string> GetPlistProperty(string plistPath, string propertyName, ILog log, CancellationToken cancellationToken = default, int attempt = 1)
```

Comment:
Not a big issue, but passing attempt as an argument while having the max attempts defined in the method is a weird decomposition as the logic responsible for doing the retry loop is split in two parts. Either pass the max attempts together with the attempt and handle the retry logic in the caller or do it all here, but the logic should be at one place.

Reply:
Good point, I was caught bein lazy